docs/asciidoc/alloc-metrics.adoc

```diff
@@ -3,12 +3,14 @@
 [width="100%",options="header"]
 |=======
 | metric name | type | description
-| reactor.netty.bytebuf.allocator.used.heap.memory | Gauge | The number of the bytes of the heap memory
-| reactor.netty.bytebuf.allocator.used.direct.memory | Gauge | The number of the bytes of the direct memory
+| reactor.netty.bytebuf.allocator.used.heap.memory | Gauge | The number of bytes committed to heap buffer allocator
+| reactor.netty.bytebuf.allocator.used.direct.memory | Gauge | The number of bytes committed to direct buffer allocator
```

Comment:
how about "reserved by" rather than "committed to"?

```suggestion
| reactor.netty.bytebuf.allocator.used.heap.memory | Gauge | The number of bytes reserved by heap buffer allocator
| reactor.netty.bytebuf.allocator.used.direct.memory | Gauge | The number of bytes reserved by direct buffer allocator
```

if accepted, please reflect this change in wording in the various other files (javadocs)